modules/shared/partials/acid-transactions.adoc

```diff
@@ -225,22 +228,16 @@ As with `TransactionFailed` it is recommended that it at least writes any logs f
 It may wish to retry the transaction at a later point, or globally extend transactional expiration times to give the protocol additional time to resolve the ambiguity.
 
 === TransactionResult.unstagingComplete()
+This boolean flag indicates whether all documents were able to be unstaged (committed).
 
-As above, there is a 'single point of truth' for a transaction.
-After this atomic commit point is reached, the documents themselves will still be individually committed (we also call this "unstaging").
-However, transactionally-aware actors will now be returning the post-transaction versions of the documents, and the transaction is effectively fully committed to those actors.
-
-So if the application is solely working with transaction-aware actors, then the unstaging process is optional.
-And failures during the unstaging process are not particularly important, in this case.
-(Note the asynchronous cleanup process will still complete the unstaging process at a later point.)
-
-Hence, many errors during unstaging will cause the transaction to immediately return success.
-That is, successful return simply means that the commit point was reached.
+For most use-cases it is not an issue if it is false.
+All transactional actors will still all the changes from this transaction, as though it had committed fully.
```

Comment:
"still see" perhaps?  seems to be missing a word.

Reply:
Will fix, thanks